ios/KaMPStarteriOS/ViewController.swift

```diff
@@ -9,12 +9,12 @@
 import UIKit
 import shared
 
-class ViewController: UIViewController, UITableViewDelegate, UITableViewDataSource {
+class ViewController: UIViewController {
 
     @IBOutlet weak var breedTableView: UITableView!
-    var data:[Breed] = []
+    var data: [Breed] = []
     
-    let log = KoinIOS().get(objCClass: Kermit.self, parameter: "ViewController") as! Kermit
+    let logger = KoinIOS().get(objCClass: Kermit.self, parameter: "ViewController") as? Kermit
```

Comment:
I liked this as a force-unwrap, because if it's null it indicates a severe enough config issue that you'd want to see the crash so you don't miss it. Otherwise, if something goes wrong, the logs silently get dropped and it's hard to tell why. It's pretty unlikely that you'd miss the crash in development and let it get to a user, but you might miss the dropped logs.

But if Swift style is strongly against that I could understand.

Reply:
It's definitely a personal preference. I, and most Swift developers I've worked with, usually go out of my/their way to avoid force unwraps. Now that you mention it though, this is kind of a different case. 

I was in a thought mode where "I don't want to crash because the logger's not there" but that's really more appropriate in an app that's bound for production. Other devs who clone this repo are our end users, so maybe if the logger's not working we _do_ want to crash.